src/app/shared/search/search.component.ts

```diff
@@ -362,6 +363,7 @@ export class SearchComponent implements OnInit {
       this.useCachedVersionIfAvailable,
       true,
       followLink<Item>('thumbnail', { isOptional: true }),
+      followLink<SubmissionObject>('item', { isOptional: true }, followLink<Item>('thumbnail', { isOptional: true })) as any,
```

Comment:
I believe this is the problematic change that is causing e2e tests to fail... It results in URLs that look like this:

```
http://localhost:8080/server/api/discover/search/objects?sort=score,DESC&page=0&size=10&configuration=default&query=test&embed=thumbnail&embed=item/thumbnail
```

The issue is the "item/thumbnail" in that URL path, as the slash is NOT URL encoded.  This means that Cypress is treating it as a subpath (and there may be a risk that other clients might do the same).

I think we need to find a way to URL encode the slash so that the param is `embed=item%2Fthumbnail`.  (Our REST API responds the same way whether the slash is encoded or not.)